next version of carbon-addons-iot-react cannot be used downstream

### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

 `Cannot find module 'carbon-addons-iot-react' from 'CommonDashboard.jsx'`

This is because the package.json and build outputs don't match anymore.  This was injected by the `monorepo` refactor.

--- a/packages/react/package.json
+++ b/packages/react/package.json
@@ -3,8 +3,8 @@
   "engines": {
     "node": "12.x || 14.x"
   },
-  "main": "lib/src/index.js",
-  "module": "es/src/index.js",
+  "main": "lib/packages/react/src/index.js",
+  "module": "es/packages/react/src/index.js",
   "unpkg": "umd/carbon-addons-iot-react.js",
   "repository": {
     "type": "git",

